### ISSUE TITLE
TOra forgets ORACLE_HOME

### ISSUE THREAD

**Issue by Blandinium** [tora-tool/tora#115]
**Steps to reproduce:**
On a Windows system with TOra3 v3.2-141-gb7ccf117 and without a ORACLE_HOME environment variable:
- Set ORACLE_HOME in "Preferences" -> "Global Settings"
- Close TOra and restart it, verify that ORACLE_HOME is still set under "Global Settings"
- Now close TOra a second time, and again restart it. The ORACLE_HOME setting will now be empty and Oracle connectivity will no longer work.

**The background:**
I depend on a number of 32 bit applications that need the 32bit Oracle driver to work. When I set a system variable ORACLE_HOME winch points to a 64bit Oracle client, TOra works, but these other applications break. I tried creating a bat file to start TOra in which I set the ORACLE_HOME variable:
```
@echo off
setlocal 
  set ORACLE_HOME=C:\MyOracleFolder && C:\MyTOraFolder\Tora.exe
endlocal
```
But TOra doesn't seem to pick up ORACLE_HOME when defined this way. Setting it in the TOra preferences works, but on a restart TOra forgets this setting again (as described above).

**Comment by ibre5041** [comment]
fixed. try the last version.

**Comment by Blandinium** [comment]
The issue is indeed fixed in Version v3.2-146-g4da93df1. Thanks!
